frontends/mit-open/src/pages/FieldPage/FieldPageSkeleton.tsx

```diff
@@ -209,11 +214,11 @@ const FieldSkeletonProps: React.FC<FieldSkeletonProps> = ({
                     flexDirection="row"
                     alignItems="end"
                     sx={{
```

Comment:
Should be some more space here - `margin-top: 8px` and `margin-bottom: 48px` seemed to look ok to me

Reply:
is this in the mobile view? also i was wondering if you had text populated in the banner. when i add this locally it seems to look like too much of a gap

Reply:
Yup - this is with data in there as well (I copied the config for OCW out of RC). Here's a screenshot:

![image](https://github.com/mitodl/mit-open/assets/945611/53565fa5-81d6-4cee-896c-e6fd2cf6dfcc)

(The white lines here are from xScope.)